libmultilabel/nn/metrics.py

```diff
@@ -253,10 +255,19 @@ def get_metrics(metric_threshold, monitor_metrics, num_classes, top_k=None):
     return MetricCollection(metrics, compute_groups=False)
 
 
-def tabulate_metrics(metric_dict, split):
+def tabulate_metrics(metric_dict: dict[str, float], split: str) -> str:
+    """Convert a dictionary of metric values into a pretty formatted string for printing.
+
+    Args:
+        metric_dict (dict[str, float]): A dictionary of metric values.
+        split (str): Name of the data split.
+
+    Returns:
+        str: Pretty formatted string.
+    """
     msg = f'====== {split} dataset evaluation result =======\n'
     header = '|'.join([f'{k:^18}' for k in metric_dict.keys()])
-    values = '|'.join([f'{x:^18.4f}' if isinstance(x, (np.floating,
+    values = '|'.join([f'{x * 100:^18.4f}' if isinstance(x, (np.floating,
```

Comment:
why this change? we cannot multiply loss by 100

Reply:
please see 

$ git diff -r 1647421915f701cb3a2a0b8038dc35ab267c4dc4 libmultilabel/nn/metrics.py 
diff --git a/libmultilabel/nn/metrics.py b/libmultilabel/nn/metrics.py
index 48afcdc..b77c721 100644
--- a/libmultilabel/nn/metrics.py
+++ b/libmultilabel/nn/metrics.py
@@ -256,7 +256,7 @@ def get_metrics(metric_threshold, monitor_metrics, num_classes, top_k=None):
 def tabulate_metrics(metric_dict, split):
     msg = f'====== {split} dataset evaluation result =======\n'
     header = '|'.join([f'{k:^18}' for k in metric_dict.keys()])
-    values = '|'.join([f'{x * 100:^18.4f}' if isinstance(x, (np.floating,
+    values = '|'.join([f'{x:^18.4f}' if isinstance(x, (np.floating,
                       float)) else f'{x:^18}' for x in metric_dict.values()])
     msg += f"|{header}|\n|{'-----------------:|' * len(metric_dict)}\n|{values}|\n"
     return msg

You can check with Shao-Syuan/Eleven about reasons of the change